app/services/process_submission_service.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Don't think this is needed if we don't use the constants

Reply:
we do now after you comment below